### ISSUE TITLE
laragon not start up until login. 

### ISSUE THREAD

**Issue by myaser55** [leokhoa/laragon#635]
im using laragon 6.0
![Capture](https://github.com/leokhoa/laragon/assets/153372433/38225542-938c-4998-93c8-9128b9ddb907)
and still not start up with windows (win server datacenter 2022)

**Comment by brahimjid** [comment]
same here

**Comment by alxndr-w** [comment]
That's right, i guess the label text for the checkbox is misleading. It will run after login like any other user software (e.g. OneDrive, Teams, ...).

![image](https://github.com/user-attachments/assets/c7f98769-605d-49ea-83ab-23c96fc5ec3e)
